Sources/Client/Client/Client.swift

```diff
@@ -203,6 +203,21 @@ public final class Client {
             webSocket.disconnectInBackground()
         }
     }
+    
+    public func subscribe(forEvents eventTypes: Set<EventType> = Set(EventType.allCases), channelId: String? = nil, callback: @escaping OnEvent) -> Subscription {
+        let subscription = Subscription { [unowned self] uuid in
+            self.onEventObservers[uuid] = nil
```

Comment:
Maybe better:
```suggestion
            self.onEventObservers.removeValue(forKey: uuid)
```

Reply:
`removeValue` has O(n) complexity, we don't need it